### DISCUSSION TITLE
Expanding the quoted_t to arrays/nested arrays

### DISCUSSION THREAD

**Discussion by raddy** [stephenberry/glaze#261] [Q&A] [answered]
If instead our buffer was something like:

std::string buffer = R"({"x":["999.2", "1015.2"]})";
.. or
std::string buffer = R"({"x":[["999.2", "1015.2"], ["12.34", "56.789"]]})";

Is there a good way to leverage quoted_t to parse these directly to doubles?

**Comment by stephenberry** [comment]
This is a great question. Because it would be best to parse these directly into something like `std::vector<double>`. I'm thinking of adding `quoted_t` to the glaze library and add support for array-like types to have quoted values. So, you would be able to register a `std::vector<double>` as an array of quoted values in the same way you could a single `double`.

**Comment by stephenberry** [comment, marked as answer]
@raddy, support has been added with #263 

You can now register higher level objects and arrays and the underlying values will be quoted.

For your use case, see how the `std::vector` is registered:

```c++
struct A
{
   double x;
   std::vector<uint32_t> y;
};

template <>
struct glz::meta<A>
{
   static constexpr auto value = object("x", glz::quoted<&A::x>(), "y", glz::quoted<&A::y>());
};
```

**Comment by raddy** [comment]
This is quite sick! 

Even this case works `std::string buffer = R"({"x":[["999.2", "1015.2"], ["12.34", "56.789"]]})";`

May want to add the nested example to your tests (since it is pretty common to stumble into as well!)

**Comment by stephenberry** [comment]
Yeah, we'll have to make more test cases, but this should even work when values are in objects as well.
